src/core/screen.c

```diff
@@ -749,9 +749,8 @@ list_windows (MetaScreen *screen)
       else
         {
 	  info->xwindow = children[i];
+          result = g_list_prepend (result, info);
```

Comment:
Indentation...

Reply:
seems to be ok here:

![2019-02-27_18-04](https://user-images.githubusercontent.com/7734191/53508531-3b04a900-3aba-11e9-8f74-c3949ad40cf5.png)

I use spaces, in the code in some lines have [spaces], in others [tab+spaces], it seems to be randomly in the function

Reply:
You are right, above line is wrong, but that is not related to your change. :) Might be good idea to fix that and remove tabs in separate pull request.